gs-academy/README.md

```diff
@@ -0,0 +1,46 @@
+
+2. Standard - There is only one training currently that is "standard" which is the `Working with Giant Swarm` introduction. This training introduces support model, communication expectations, and high level view. This is typically done as part of onboarding or when customers have new joiners or teams added to the GS platform. Newly created and implemented was the second part of the kick-off training `Getting started with kubectl-gs` which is meant to learn about access and getting a deeper look into under the hood of the GS platform. Again, this is typically scheduled by the customer's Account Engineer. 
+
+## Proposal
+
+The idea would be to implement a formal program `Giant Swarm Academy` that would encompass onboarding, training, and continuing education for both new and current customers. The main points would consist of:
+
+- All ad-hoc and standard trainings would become regular, recurring trainings at desired intervals. Still to be discussed is the interval however the idea would be every team gives a their training once a month and the schedule is staggered. 
+
+```
+# Example
+First tuesday of every month - "Working with GS" and "Getting started with Kubectl GS"
+First thursday of every month - Kube 101
+Second tuesday of every month - Security 101
+Second thursday of every month - Gitops 101
+...
+```
+
+- All trainings should include an element of hands-on game or practical application of the concepts. Examples may be kube 101, everyone stands up a local minecraft server or security everyone rebuilds an image to be more secure. This encourages practice on the technical concept but also practice working with each other.  
+
+- We would build specifical certificates for linkedin and/or POAPs (proof of attendance protocol) style NFTs. This would enable people to show their achievements and allow the possibiliy of different knowledge levels based on the course such as `Foundations`, `Practioner`, and `Expert` as an example when moving on to more advanced sessions.  
+
+## Open Questions
+
+- Training schedule. What should the intervals be and what is the effort for the teams?
+
+- Are certain trainings needed more than other trainings and should be offered more regularly?
+
+- How many levels of trainings should each area have? Example: `Foundations`, `Practioner`, `Expert`
+
+- Shold these trainings be customer specific or cross customer? Community and openess is always a plus but we also want to make sure that all trainees feel that they are in a comfortable enough enviornment to ask all questions. The other point is that not all customers are at the same expertise level in these areas. One idea for mixed customers is using an anonymous question system.  
+
+- What should the proof of achievement be?
```

Comment:
LinkedIn-badges is what is widely used also by CNCF so using sth familiar would be my suggestion.